bundles/org.eclipse.equinox.preferences/src/org/eclipse/core/runtime/preferences/IScopeContext.java

```diff
@@ -29,6 +29,34 @@
  */
 public interface IScopeContext {
 
+	/**
+	 * Property used when a scope context is registered as a service to distinguish
+	 * different scopes
+	 *
+	 * @since 3.9
```

Comment:
This is most likely wrong. 3.9 version was released already, we can't add API with old version numbers.

I see that API filters were added, because API tooling wants to increase **major** bundle version, but this doesn't mean if we don't increase major we won't increase minor segment.

This causes such errors like https://github.com/eclipse-platform/eclipse.platform.resources/issues/102#issuecomment-1112987841

Because client code that receives this API re-exported via org.eclipse.core.runtime doesn't see new fields added if the installation contains "old" 3.9 bundles without this API.

So I think we must:
1) bump org.eclipse.equinox.preferences to 3.10 here : https://github.com/eclipse-equinox/equinox.bundles/blob/f3c4cedb47cf2cfc007b55161f1954fd01d55551/bundles/org.eclipse.equinox.preferences/META-INF/MANIFEST.MF#L5
2) also bump version **of the package** (see manifest that says it has 3.4.0 version: https://github.com/eclipse-equinox/equinox.bundles/blob/f3c4cedb47cf2cfc007b55161f1954fd01d55551/bundles/org.eclipse.equinox.preferences/META-INF/MANIFEST.MF#L13) 
3) bump re-exported package in org.eclipse.core.runtime to 3.10 here https://github.com/eclipse-platform/eclipse.platform.runtime/blob/4cc1d01f2d8f2e9494b69003f05e064683ebc7cd/bundles/org.eclipse.core.runtime/META-INF/MANIFEST.MF#L16